orca-core/src/main/java/com/netflix/spinnaker/orca/pipeline/model/support/TriggerDeserializer.kt

```diff
@@ -138,7 +138,8 @@ internal class TriggerDeserializer :
           get("source").textValue(),
           get("project").textValue(),
           get("branch").textValue(),
-          get("slug").textValue()
+          get("slug").textValue(),
+          get("action").textValue()
```

Comment:
think we'll need to supply a default value here or make action on GitTrigger nullable

would also be good to ensure there is a test case that attempts deserialization with no supplied action

Reply:
thanks for feedback!